ExtractorUtils.Test/CDFTester.cs

```diff
@@ -50,11 +52,23 @@ public CDFTester(string[] config, ITestOutputHelper output)
             DestinationWithIDM = Provider.GetRequiredService<CogniteDestinationWithIDM>();
             Prefix = TestUtils.AlphaNumericPrefix("net-utils-test-");
             Source = new CancellationTokenSource();
-            SpaceId = $"dotnet-extractor-utils-test-space{i}";
+            _spaceId = $"dotnet-extractor-utils-test-space{i}";
         }
         public CDFTester(CogniteHost host, ITestOutputHelper output) : this(GetConfig(host), output)
         {
-            DestinationWithIDM.CogniteClient.DataModels.UpsertSpaces(new List<SpaceCreate>() { new() { Space = SpaceId } }).GetAwaiter().GetResult();
+        }
+
+        public async Task<string> GetSpaceId()
```

Comment:
should this be synchronized to avoid concurrent loads? Not a big deal since only test code